modules/mri_violations/templates/menu_mri_protocol_violations.tpl

```diff
@@ -107,11 +107,7 @@
                     {foreach from=$mpgroup item=protocol}
                         <tr>
                         {foreach from=$protocol key=k item=v}   
-                            {if $violated_scans_modifications && $k != 'Protocol Group'}
-                            <td id="row_{$protocol.ID}_td_{$k}" class='description' contenteditable = "true" nowrap {$rowBorder}>
-                            {else}
```

Comment:
Can you delete the other code too?

1. The script in the ajax directory is now unnecessary
2. The js in `js/mri_protocol_violations.js` related to this feature

Reply:
Done.